README.md

```diff
@@ -1,18 +1,38 @@
 # Managed Structure Template
-This template is a starting point for creating programs that can run as Griptape Cloud Managed Structures. 
+
+This template is a starting point for creating programs that can run as Griptape Cloud Managed Structures.
 It provides a basic project layout for creating a Managed Structure, but you can customize it to fit your needs.
 
 See the [Griptape CLI](https://github.com/griptape-ai/griptape-cli?tab=readme-ov-file#skatepark-emulator) instructions for how to get started with this template.
 
 ## Project Structure
+
 The template provides the following project layout:
 
 - `structure.py` Contains the Managed Structure code.
 - `requirements.txt` Contains the dependencies for the Managed Structure. These will be automatically installed when the Structure is registered with Skatepark.
-- `example-client/client.py` Contains an example client that _uses_ the Managed Structure's API. This is useful for testing your Managed Structure locally but ultimately you will want to integrate your Managed Structure with your own application. 
+- `structure_file.yaml` Contains the Managed Structure configuration required by Skatepark and Griptape Cloud to properly build and run the Structure.
```

Comment:
`structure_file` is vague, could we make it `structure_config.yaml` or `structure_configuration.yaml`?

Reply:
I think calling it "Structure Config" is confusing, since we already have [StructureConfig](https://github.com/griptape-ai/griptape/blob/e1977ce2f42665221482a1d365cc92931c5c6179/griptape/config/structure_config.py#L28) in the framework, which represents something different.

`structure_file.yaml` is similar naming to files used by other build systems, like Makefile, Dockerfile, etc (though it has a more python like file name, and the extension is nice for ide syntax highlighting)

Reply:
I think that's splitting hairs, as technically our concept of Managed Structures isn't tied to the framework Structure class at all, anyway. And I would argue that `structure_file` could just as easily be interpreted to be the main file for the structure.

Reply:
The backend does not have a hard requirement on the name of the file itself, so this can be named whatever we think will make the most sense. I do see both points about how `structure_config` and `structure_file` could be cause for confusion.

I'll update it to `structure_configuration.yaml` unless we find a better name.

Reply:
If we don't care about the confusion, I'll vote for `structure_config.yaml` 

(`structure_configuration.yaml` just looks too long to me)

Reply:
I also like `structure_config`, though is it confusing with the framework `StructureConfig`?

Reply:
Updated to structure_config.yaml